### PR TITLE
Initial D support

### PR DESCRIPTION
**Pull request by andre2007**
This is a raw implement to get initial support for D
Some remarks:

- If a module is named fx, then it is not allowed to have a function also name fx. Therefore I named the function executeFx

- In further version, the signature of function executeFx can be analyzed during compile time and clients would be able to write executeFx(int a, int b) instead of dealing with JSON. But thats sugar on top

At the moment I have some issue to build fx at all on windows. Therefore the coding is not really tested

### CODE REVIEW

File: assets/dockerfiles/fx/d/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM metrue/fx-d-base
```
Comment (metrue) [inline assets/dockerfiles/fx/d/Dockerfile]: @andre2007 

I already pushed the image generated by your initial Dockerfile in assets/dockerfiles/base/d to Docker Hub already.

File: assets/dockerfiles/fx/d/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
```
Comment (metrue) [inline assets/dockerfiles/fx/d/Dockerfile]: seems like you're missing adding your source files here.
```
ADD . .
```
should fix.

Comment (andre2007) [reply assets/dockerfiles/fx/d/Dockerfile]: I added the COPY . . command as all other dockerfiles also using this command

**Comment by metrue** [comment]
@andre2007 

I think your PR is ready to merge now, the testing fails is not relate to this change, I fix that soon.

**Pull request merged**